WorkbenchProject/src/main/java/edu/uwb/braingrid/workbench/provvisualizer/controller/ProVisCtrl.java

```diff
@@ -153,6 +155,19 @@ public void handle(long now) {
         };
 
         timer.start();
+
+        // Add an event handler to the primary stage to stop the timer when the application is minimized (iconified),
+        // and start it again when it is no longer minimized.
```

Comment:
This comment should wrap at 100 characters. Currently causes a checkstyle error.

Reply:
Strange, I know I got that error when building, and I thought I fixed it since my other builds succeeded. Trying to build it now (with those line you selected as is) fails, I must have missed something before.